build_tools/pkgci/external_test_suite/sdxl_scheduled_unet_gpu_rocm.json

```diff
@@ -0,0 +1,32 @@
+{
+    "config_name": "gpu_rocm",
+    "iree_compile_flags" : [
+      "--iree-hal-target-backends=rocm",
+      "--iree-rocm-target-chip=gfx90a",
```

Comment:
MI250? (anything specific to a particular device should call that out in the file name, so we can add other devices without name conflicts)